modules/vaos/app/controllers/vaos/v2/facilities_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
                                                           schedulable:,
                                                           children:,
                                                           type:)
+        response[:data] = sort_by_recent_facilities(response[:data]) if params[:sort_by] == 'recentLocations'
```

Comment:
@simiadebowale I've deviated a bit from the ticket regarding naming. I see all other parameters are snake_cased so I changed `sortBy` into `sort_by` for the parameter name. Alternatively, I've seen other teams use `sort` as well. I've kept `recentLocations` as the value since there doesn't seem to be any specific pattern on what values should be cased as.